src/pyfaf/actions/repoimport.py

```diff
@@ -99,8 +101,8 @@ def run(self, cmdline, db):
                                .format(repo.name))
                 return 1
 
-            self.log_info("Adding repository '{0}' ({1})"
-                          .format(repo.name, repo.url))
+            self.log_info("Adding repository '{0}' {1}"
+                          .format(repo.name, [url.url for url in repo.url_list]))
```

Comment:
Nit: What about to convert the list of URLs to string? The default conversion looks terrible.